src/main/java/seedu/address/model/util/DateTime.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.model.util;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.TemporalAccessor;
+
+public class DateTime implements Comparable<DateTime> {
```

Comment:
Would optional DateTime be supported?

Reply:
My impression is that classes deadline, to and from would have an Optional<DateTime> type value instead. So while dateTime does not take in an optional string, classes that have DateTime should handle empty strings before parsing a DateTime.